Examples/AsposeImagingCloudSDKExamples/AsposeImagingCloudSDKExamples.csproj

```diff
@@ -33,9 +33,8 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Aspose.Imaging.Cloud.Sdk, Version=19.6.0.0, Culture=neutral, PublicKeyToken=70f46a67678f77f1, processorArchitecture=MSIL">
-      <HintPath>..\packages\Aspose.Imaging-Cloud.19.6.0\lib\net45\Aspose.Imaging.Cloud.Sdk.dll</HintPath>
-      <Private>True</Private>
+    <Reference Include="Aspose.Imaging.Cloud.Sdk, Version=19.10.0.0, Culture=neutral, PublicKeyToken=70f46a67678f77f1, processorArchitecture=MSIL">
+      <HintPath>..\packages\Aspose.Imaging-Cloud.19.10.0\lib\net45\Aspose.Imaging.Cloud.Sdk.dll</HintPath>
     </Reference>
     <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
why not use Version=19.12.0.0 ?

Reply:
We use Aspose.Imaging-Cloud from nuget. It can be updated only after publishing v19.12